pom.xml

```diff
@@ -657,55 +685,58 @@
                                             setClientVersion('new.ignite.version.fixed', 'new.client.version');
                                         </script>
 
-                                        <echo message="Update ignite.version in .NET" />
-                                        <echo message="${new.client.version}" />
+                                        <echo message="Update ignite.version in .NET"/>
```

Comment:
These changes not about protocol change. Could you please revert it? You can submit it in a separate issue.